mods/cnc/maps/shellmap/shellmap.lua

```diff
@@ -1,3 +1,248 @@
+CommandoTeam = {"rmbo", "rmbo"}
+OrcaSquadron = {"orca", "orca", "orca", "orca"}
+
+LSTREntryPath = {LSTRStart.Location, LSTRMove1.Location, LSTRMove2.Location, LSTRLand.Location}
+LSTMEntryPath = {LSTMStart.Location, LSTMMove1.Location, LSTMMove2.Location, LSTMLand.Location}
+LSTLEntryPath = {LSTLStart.Location, LSTLMove1.Location, LSTLMove2.Location, LSTLLand.Location}
+
+LSTRExitPath = {LSTRMove2.Location + CVec.New(0,1), CPos.New(8,LSTRStart.Y)}
+LSTMExitPath = {LSTMMove2.Location + CVec.New(0,1), CPos.New(8,LSTMStart.Y)}
+LSTLExitPath = {LSTLMove2.Location + CVec.New(0,1), CPos.New(8,LSTLStart.Y)}
+
+--DEBUG Functions: Remove when Shellmap deemed "complete"
+SkipCommandos = function()
+	Turret1.Destroy()
+	Turret2.Destroy()
+	Turret3.Destroy()
+	Turret4.Destroy()
+	beachdude1.Destroy()
+	beachdude2.Destroy()
+	Beachhead()
+end
+--DEBUG End
+
+MakeUnitInvincible = function(unit)
+	if unit.HasProperty("AcceptsUpgrade") and unit.AcceptsUpgrade("unkillable") then
+		unit.GrantUpgrade("unkillable")
+		unit.Stance = "Defend"
+	end
+end
+
+WakeNodRefn = function()
+	refnHand = Reinforcements.Reinforce(nod, { "hand" }, { (HandBuild.Location-CVec.New(0,1)) })[1]
+	Trigger.OnDamaged(refnHand, function(a) a.StartBuildingRepairs(a.Owner) end)
+
+	--todo: Nod outpost does more things
+end
+
+WakeNodMain = function()
+	--todo: Big Nod Base starts doing things
+end
+
+GDIStartInfantry = function()
+	--todo: GDI starts making infantry
+end
+
+GDIStartVehicles = function()
+	--todo: GDI starts making vehicles
+end
+
+SendCommandos = function()
+	local commandos = Reinforcements.ReinforceWithTransport(gdi, "tran", CommandoTeam, 
+	{ChinEntry.Location, ChinUnload.Location}, {ChinExit.Location})[2]
+	
+	Trigger.OnIdle(commandos[1], function()
+			if(Map.NamedActor("Turret1") ~= nil) then
+				commandos[1].GoDemolish(Turret1)
+			end
+			if(Map.NamedActor("beachdude1") ~= nil or Map.NamedActor("beachdude2") ~= nil) then
```

Comment:
I'm not quite sure what you're referring to. I send each commando at two different turrets.